### PR TITLE
compiling on v1.0

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
@Vaibhavdixit02

**Comment by Vaibhavdixit02** [comment]
@xukai92 I am facing problem with adding Turing as a dependency in Julia 0.7, is this a known issue? I saw a 0.5.0 pre-release which is supposed to work on 1.0 but it is not available on 0.7?

File: REQUIRE [hunk 1]
```diff
@@ -3,7 +3,7 @@ DiffEqBase 1.7.0
 #Mamba
 Stan
 Distributions
-Turing 0.5.0
+Turing 0.4.4
```
Comment (ChrisRackauckas) [inline REQUIRE]: no, revert this.

**Comment by ChrisRackauckas** [comment]
It was working fine locally. The issue may be the lower bound to the v0.4.0 + Pkg's enforcement of semvar. Don't make a lower bound lower than necessary: v0.5.0 of Turing.jl is required for v0.7.

**Comment by Vaibhavdixit02** [comment]
v0.5.0 is not available on 0.7. Can you comfirm which version you used
locally?

**Comment by ChrisRackauckas** [comment]
https://github.com/TuringLang/Turing.jl/blob/v0.5.0/REQUIRE

**Comment by ChrisRackauckas** [comment]
https://github.com/JuliaLang/METADATA.jl/blob/metadata-v2/Turing/versions/0.5.0/requires

**Pull request merged**